### PR TITLE
Adding shortcode [gp-project-contributors-translators]

### PR DESCRIPTION
**Pull request by itssimple**
Creates a table that shows what users that have translated anything for the locales.
Currently shows current number of translations (Tooltip for stats)

### CODE REVIEW

File: gp-project-contributors.php [hunk 1]
```diff
@@ -112,6 +113,132 @@ public function gp_project_contributors( $atts ) {
...
+{$gp_table_prefix}translation_sets.*
+FROM {$gp_table_prefix}translations
+INNER JOIN {$gp_table_prefix}translation_sets ON {$gp_table_prefix}translations.translation_set_id = {$gp_table_prefix}translation_sets.id
+INNER JOIN `{$wpdb->users}` on `{$gp_table_prefix}translations`.`user_id` = `{$wpdb->users}`.`ID`
+INNER JOIN {$gp_table_prefix}originals ON {$gp_table_prefix}translations.original_id = {$gp_table_prefix}originals.id
+WHERE `{$gp_table_prefix}originals`.`project_id` = {$project_id}
+AND `{$gp_table_prefix}originals`.`status` = '+active'
+GROUP BY {$wpdb->users}.user_login, {$gp_table_prefix}translation_sets.id" );
+		
+		// $return .= '<pre>' . print_r( $result, true ) . '</pre>';
```
Comment (toolstack) [inline gp-project-contributors.php]: Delete this as it's not used.

File: gp-project-contributors.php [hunk 2]
```diff
@@ -112,6 +113,132 @@ public function gp_project_contributors( $atts ) {
...
+			$contribs->waiting = $row->waiting_contrib;
+			$contribs->old = $row->old_contrib;
+			$contribs->rejected = $row->rejected_contrib;
+			
+			$contribs->total = $row->total_contrib;
+			$contribs->tooltip = "Current: " . number_format_i18n($contribs->current, 0) . "\r\n" .
+				"Fuzzy: " . number_format_i18n($contribs->fuzzy, 0) . "\r\n" .
+				"Waiting: " . number_format_i18n($contribs->waiting, 0) . "\r\n" .
+				"Old: " . number_format_i18n($contribs->old, 0) . "\r\n" .
+				"Rejected: " . number_format_i18n($contribs->rejected, 0);
```
Comment (toolstack) [inline gp-project-contributors.php]: The Fuzzy/Waiting/Old/Rejected strings should be translated with `__()` instead of hard coded.

**Review by toolstack — CHANGES_REQUESTED** [review]
Look good except for a couple of minor items.

File: gp-project-contributors.php (hunk 1 above)
Comment (itssimple) [reply gp-project-contributors.php]: Removed

File: gp-project-contributors.php (hunk 2 above)
Comment (itssimple) [reply gp-project-contributors.php]: Added i18n-code to the items

File: gp-project-contributors.php [hunk 3]
```diff
@@ -112,6 +113,130 @@ public function gp_project_contributors( $atts ) {
...
+			$contribs->old = $row->old_contrib;
+			$contribs->rejected = $row->rejected_contrib;
+			
+			$contribs->total = $row->total_contrib;
+			$contribs->tooltip = __("Current") . ": " . number_format_i18n($contribs->current, 0) . "\r\n" .
+				__("Fuzzy") . ": " . number_format_i18n($contribs->fuzzy, 0) . "\r\n" .
+				__("Waiting") . ": " . number_format_i18n($contribs->waiting, 0) . "\r\n" .
+				__("Old") . ": " . number_format_i18n($contribs->old, 0) . "\r\n" .
+				__("Rejected") . ": " . number_format_i18n($contribs->rejected, 0);
+
```
Comment (toolstack) [inline gp-project-contributors.php]: Add the plugin translation domain as well 'gp-project-contributors' to the translation calls.

Comment (itssimple) [reply gp-project-contributors.php]: Right, sorry. :)

**Review by toolstack — APPROVED** [review]

**Pull request merged**